### ISSUE TITLE
METADATA: Unknown types found, no types for ValidationCodeHash

### ISSUE THREAD

**Issue by hitchhooker** [canontech/staking-payouts#65]
getting following error with latest version after exiting claiming payouts
```
kusamaxi@vmi560015:~$ cat crontab.logs.docker.kusamaxi
2021-06-25 15:42:10 [payouts] info: No payouts to claim
2021-06-25 15:42:10 [payouts] info: Exiting ...
2021-06-25 15:42:26          API-WS: disconnected from wss://kusama-rpc.polkadot.io: 1006:: Connection dropped by remote peer.
2021-06-25 15:42:31        METADATA: Unknown types found, no types for ValidationCodeHash
2021-06-25 15:42:32        API/INIT: Error: FATAL: Unable to initialize the API: createType(ValidationCodeHash):: Cannot construct unknown type ValidationCodeHash
    at EventEmitter.value (/app/node_modules/@polkadot/api/base/Init.cjs:88:25)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
2021-06-25 15:43:29          API-WS: disconnected from wss://kusama-rpc.polkadot.io: 1006:: Connection dropped by remote peer.
2021-06-25 15:43:36        METADATA: Unknown types found, no types for ValidationCodeHash
2021-06-25 15:43:36        API/INIT: Error: FATAL: Unable to initialize the API: createType(ValidationCodeHash):: Cannot construct unknown type ValidationCodeHash
    at EventEmitter.value (/app/node_modules/@polkadot/api/base/Init.cjs:88:25)
    at processTicksAndRejections (node:internal/process/task_queues:94:5)
```

**Comment by emostov** [comment]
This issue was fixed with 1.2.1 - can you please make sure you are using that exact version, have installed its dependencies and rebuilt?

**Comment by emostov** [comment]
Closing due to inactivity
